step6/index.py

```diff
@@ -71,6 +71,63 @@ def info(self):
 dog.set_owner('Donald Trump')
 print(dog.get_owner())
 
+# Abstract Class
+from abc import ABCMeta, abstractmethod
```

Comment:
@osama10 could you please write some description on ` Abstract Base Classes (ABCs)` for beginners

Reply:
will do .